backend/src/main/java/ai/verta/modeldb/configuration/AppConfigBeans.java

```diff
@@ -156,6 +155,36 @@ public ServerBuilder<?> serverBuilder(
   }
 
   @Bean
+  public MigrationSetupConfig migrationSetupConfig(MDBConfig mdbConfig) {
+    return new MigrationSetupConfig(mdbConfig);
+  }
+
+  @Bean
+  public Migration migration(MigrationSetupConfig migrationSetupConfig) throws Exception {
+    return new Migration(migrationSetupConfig);
+  }
+
+  @Bean
+  // isRunSeparateMigration is true then execute below function
+  public CommandLineRunner commandLineRunner(
+      Migration migration, MigrationSetupConfig migrationSetupConfig) {
+    return args -> {
+      migration.migrate();
+      LOGGER.info("Migrations have completed");
+
+      var runLiquibaseSeparate = migrationSetupConfig.isRunSeparateMigration();
+      LOGGER.trace("run Liquibase separate: {}", runLiquibaseSeparate);
+      if (runLiquibaseSeparate) {
+        LOGGER.info("System exiting");
+        this.appContext.initiateShutdown(0);
+        System.exit(0);
```

Comment:
It will shutdown gracefully first and then we will exit the system

Reply:
I'm surprised something called "initiateShutdown" blocks until the shutdown is complete. /shrug

Reply:
`initiateShutdown` is our own method which called `SpringApplication.exit(this.applicationContext, () -> returnCode)` so it is also run on main thread so it will block and then at last `System.exit(0);` will call.

Reply:
If we are not adding `System.exit(0);` then `SpringApplication.exit(this.applicationContext, () -> returnCode)` will only triggered graceful shutdown but it will not exist the system and stay hold on manually exist system so we need to add `System.exit(0);`

Reply:
@jkwatson-verta So what would you suggest for this? Which name you suggest for it?

Reply:
just `shutdown()` I guess? Let's not change it for this PR, though. Just something to think about.